scripts/lib/mkcloud-onhost.sh

```diff
@@ -145,7 +145,17 @@ function onhost_cacheclouddata
             # Now the various test images
             echo "images/$a/other/magnum-service-image.qcow2"
             echo "images/$a/other/manila-service-image.qcow2"
+            # these are the real ones, the above are just symlinks
+            echo "images/$a/other/fedora-23-atomic-7.qcow2"
+            echo "images/$a/other/manila-service-image.x86_64-0.13.0-Build14.1.qcow2"
```

Comment:
Will this work as expected? `$architectures` resolves to `aarch64 x86_64 s390x` so this would be echoing

```
"images/aarch64/other/manila-service-image.x86_64-0.13.0-Build14.1.qcow2"
"images/x86_64/other/manila-service-image.x86_64-0.13.0-Build14.1.qcow2"
"images/s390x/other/manila-service-image.x86_64-0.13.0-Build14.1.qcow2"
```

which doesnt seem right to me?

Reply:
Good catch! So far I've only tested x86_64. I was merely trying to sync the source of the symlinks for both "magnum-service-image.qcow2" and "manila-service-image.qcow2" in line 146-147. So far I don't have any aarch64 and s390x to test the changes. But I agree with you. I can't see how it will work for those two architectures.